### ISSUE TITLE
Docker build failing: ../redoctober/client/client.go:15:2: cannot find package "github.com/cloudflare/cfssl/transport/core"

### ISSUE THREAD

**Issue by tlianza** [cloudflare/cfssl#494]
In an ubuntu VM, running `script/build -os="linux" -arch="386"` gives me this output. Same with amd64:

```
Sending build context to Docker daemon 5.393 MB
Sending build context to Docker daemon
Step 0 : FROM golang:1.4.2-cross
 ---> b99ec45c9d74
Step 1 : RUN go get github.com/mitchellh/gox
 ---> Using cache
 ---> 587d193b3974
Step 2 : RUN go get github.com/tools/godep
 ---> Using cache
 ---> 5823b001f7ee
Step 3 : RUN go get github.com/cloudflare/cf-tls/tls
 ---> Using cache
 ---> ee27b3192a35
Step 4 : RUN go get github.com/cloudflare/go-metrics
 ---> Using cache
 ---> ec166fb0a6ab
Step 5 : RUN go get github.com/cloudflare/redoctober/core
 ---> Using cache
 ---> 92f2c5576bdc
Step 6 : RUN go get github.com/dgryski/go-rc2
 ---> Using cache
 ---> 0460dad97565
Step 7 : RUN go get golang.org/x/crypto/ocsp
 ---> Using cache
 ---> cef6b981b2ff
Step 8 : RUN go get github.com/GeertJohan/go.rice
 ---> Using cache
 ---> 20efcb609fe2
Step 9 : ENV GOPATH /go/src/github.com/cloudflare/cfssl:/go
 ---> Using cache
 ---> 2083e1946880
Step 10 : ENV USER root
 ---> Using cache
 ---> 953e65ceec1e
Step 11 : WORKDIR /go/src/github.com/cloudflare/cfssl
 ---> Using cache
 ---> dea4b382fc75
Step 12 : ADD . /go/src/github.com/cloudflare/cfssl
 ---> 1e0fc62ca06a
Removing intermediate container 5cd4fb420c1c
Successfully built 1e0fc62ca06a
Number of parallel builds: 1

-->       linux/386: github.com/cloudflare/cfssl/cmd/cfssl
-->       linux/386: github.com/cloudflare/cfssl/cmd/cfssljson
-->       linux/386: github.com/cloudflare/cfssl/cmd/mkbundle
-->       linux/386: github.com/cloudflare/cfssl/cmd/multirootca

1 errors occurred:
--> linux/386 error: exit status 1
Stderr: ../redoctober/client/client.go:15:2: cannot find package "github.com/cloudflare/cfssl/transport/core" in any of:
	/usr/src/go/src/github.com/cloudflare/cfssl/transport/core (from $GOROOT)
	/go/src/github.com/cloudflare/cfssl/src/github.com/cloudflare/cfssl/transport/core (from $GOPATH)
	/go/src/github.com/cloudflare/cfssl/transport/core
```

**Comment by xynova** [comment]
It is happening to me as well for the release 1.1.0

**Comment by jacobhaven** [comment]
This issue has been solved (the Dockerfiles were updated), but there is still a CGO sqlite dependency that needs to be fixed #545
